stdlib/public/core/Index.swift

```diff
@@ -92,7 +92,7 @@ public postfix func ++ <T : _Incrementable> (inout i: T) -> T {
 /// Represents a discrete value in a series, where a value's
 /// successor, if any, is reachable by applying the value's
 /// `successor()` method.
-@swift3_migration(renamed="ForwardIndex")
+@swift3_migration(message="ForwardIndex methods are now available from the Collection protocol.")
```

Comment:
Could you also add something like "In Swift 3, collection indices don't need to implement ForwardIndex, but only required to implement Comparable." to push the user toward the right migration.